crates/holochain/src/core/workflow/validation_receipt_workflow/tests.rs

```diff
@@ -10,6 +10,7 @@ use holochain_state::prelude::*;
 use rusqlite::Transaction;
 
 #[tokio::test(flavor = "multi_thread")]
+#[ignore = "flaky, doesn't take into account timing or retries"]
```

Comment:
👍🏻 Yes I believe I re-enabled this test because it works most of the time but it is flaky. It's on my list of tests I'd like to have a go at. Please could this be logged in the flaky-tests channel to be looked at?